drone/src/agent/executor.rs

```diff
@@ -400,7 +400,13 @@ impl<E: Engine> Executor<E> {
                 };
 
                 tracing::info!(%backend_addr, "Got address from container.");
-                wait_port_ready(&backend_addr).await?;
+                if let Err(_err) = wait_port_ready(&backend_addr).await {
+                    tracing::error!(
+                        "backend {} timed out before ready",
+                        spawn_request.backend_id
+                    );
```

Comment:
Can we make this a `warn` since this most likely indicates a client error. (Also [here](https://github.com/drifting-in-space/plane/blob/main/core/src/retry.rs#L23))